CREATED TEXTAREA COMPONENT

AvantikaM5 opened the pull request:
Component is auto-growing based on the user's input and send icon is relatively changing it's position

AvantikaM5 committed:
CREATED TEXTAREA COMPONENT

chef-danny-d reviewed (changes requested):
I like your implementation of useRef in combination with a React hook to calculate the sizing of the textarea. The component is still lacking flexibility so I'll be addressing these issues.

Commit:
:recycle: (PR): Made the textarea component more dynamic
Extracted most of the attributes that the element accepts into component props with default values.

chef-danny-d committed:
Merge branch 'dev' into ALMP-396
